data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,58 @@ mission "Arachi Cartoonist 2"
+			`	You hear the curtains closing, and the lights of the theater turn back on again, as the audience is taken by a round of applause. Once they've settled, the announcer goes up to the stage again. "And with that, concluded, our soiree is. Thank you all for gracing us with your presence, I, in name of the theater, do." He lowers his head in a bow, and leaves the stage again, as nearly all of the patrons in the audience get up and head for the exit, this time with nobody else coming in to take their seats. At the same time, the orchestra starts packing up their instruments. There are, however, a few who remain in their seats, including the Arach who brought you here.`
+			choice
+				`	(Stay.)`
+					goto end
+				`	(Leave.)`
+			label leaveearly
+			`	You thank the Arach beside you for inviting you here, and while she doesn't seem to pay you much mind, you can feel the gazes of other Arachi patrons on you, whispering among themselves as you leave. A few minutes after leaving, you're back at your ship.`
+				decline
+			label end
+			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. About ten minutes later, however, you hear the curtains open, as some violins start to play in unison, not from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure, but it isn't an actor, seemingly being made of some sort of wooden parts, kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with "arms" opening to the sides, and becoming even taller, as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand what they said. You do recognize, however, that it isn't being sung in any language of the Coalition.`
```

Comment:
```suggestion
			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. A couple minutes later, however, you hear the curtains open, as some violins start to play in unison, not from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure, but it isn't an actor, seemingly being made of some sort of wooden parts, kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with "arms" opening to the sides and becoming even taller as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand what they said. You do recognize, however, that it isn't being sung in any language of the Coalition.`
```
10 minutes seems like a very long time for someone to sit in a dark theater and not ask any questions. Would it be problematic to change this? If so, maybe we could add something where the player tries to ask a question and is answered with a "just wait and see".